lib/index.ts

```diff
@@ -0,0 +1,13 @@
+import ComponentManager from "./componentManager";
+
+declare global {
+  interface Window { ComponentManager: unknown }
```

Comment:
This will throw an error: `Type 'typeof ComponentManager' is missing the following properties from type 'ComponentManager': component, sentMessages, messageQueue, coallesedSaving, and 40 more.ts(2740)`.

The use of `unknown` here is to define `ComponentManager` in the `window` object.